google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/AbstractStatementParser.java

```diff
@@ -361,8 +392,58 @@ ClientSideStatement getClientSideStatement() {
   static final Set<String> dmlStatements = ImmutableSet.of("INSERT", "UPDATE", "DELETE");
   private final Set<ClientSideStatementImpl> statements;
 
+  /** The default maximum size of the statement cache in Mb. */
+  public static final int DEFAULT_MAX_STATEMENT_CACHE_SIZE_MB = 20;
```

Comment:
Why public?

Reply:
This feature is used in the Connection API, which itself is not really a public API. Instead, the JDBC driver and PGAdapter are, and these sometimes want to use this default as well, for example so they can include it in automated help text and/or the list of properties that they support.